src/artemis/tests/test_fast_grid_scan_plan.py

```diff
@@ -60,136 +64,126 @@ def test_ispyb_params_update_from_ophyd_devices_correctly():
     slit_gaps.xgap.sim_put(xgap_test_value)
     slit_gaps.ygap.sim_put(ygap_test_value)
 
-    RE(update_params_from_epics_devices(params, undulator, synchrotron, slit_gaps))
+    class TestCB(CallbackBase):
```

Comment:
this test is for the read_hardware_for_ispyb plan, not the communicator, that's also why it's in test_fast_grid_scan_plan, it wasn't supposed to test that the documents are handled correctly. I've renamed it to hopefully be more clear, but I don't quite understand what you meant by
>if it fails it may be that you need to update FGSCommunicator

Reply:
I guess what I mean is that what we're testing here is that the expected documents are created but it's not clear just from this test why we care that those specific documents are created. We care because `FGSCommunicator` needs them so either using the real `FGSCommunicator` here or having a comment as to why we care about the test would make that clearer. It's fine though, happy to leave as is.